.github/workflows/concrete_python_release_cpu.yml

```diff
@@ -399,27 +399,27 @@ jobs:
           PYTHON=${{ format('python{0}', matrix.python-version) }}
 
           # Initialize an empty test environment
-          export TEST_TMP_DIR=$(mktemp -d)
-          echo "TEST_TMP_DIR=$TEST_TMP_DIR" >> "${GITHUB_ENV}"
-          cd $TEST_TMP_DIR
+          TEST_TMP_DIR=$(mktemp -d)
+          echo "TEST_TMP_DIR=${TEST_TMP_DIR}" >> "${GITHUB_ENV}"
+          cd "${TEST_TMP_DIR}" || exit
 
           # Activate virtual environment
           $PYTHON -m venv .testenv && source .testenv/bin/activate
 
           # Install extra requirements for tests
-          pip install $WHEEL_DIR/*macos*.whl
-          pip install -r $CONCRETE_PYTHON/requirements.dev.txt
+          pip install "${WHEEL_DIR}"/*macos*.whl
+          pip install -r "${CONCRETE_PYTHON}"/requirements.dev.txt
 
           # MacOS x86 have conflict between our OpenMP library, and one from torch
           # we fix it by using a single one (from torch)
           # see discussion: https://discuss.python.org/t/conflicting-binary-extensions-in-different-packages/25332/8
-          export SITE_PACKAGES=$(pwd)/.testenv/lib/$PYTHON_VERSION/site-packages/
-          find $SITE_PACKAGES -not \( -path $SITE_PACKAGES/concrete -prune \) -name 'lib*omp5.dylib' -or -name 'lib*omp.dylib' | xargs -n 1  ln -f -s $SITE_PACKAGES/concrete/.dylibs/libomp.dylib
+          SITE_PACKAGES=$(pwd)/.testenv/lib/$PYTHON_VERSION/site-packages/
+          find . "${SITE_PACKAGES}" \( -not \( -path "${SITE_PACKAGES}"/concrete -prune \) -name 'lib*omp5.dylib' -or -name 'lib*omp.dylib' \) -exec xargs -n 1  ln -f -s "${SITE_PACKAGES}"/concrete/.dylibs/libomp.dylib {} \;
```

Comment:
I would advise not to switch to `-exec`, as there might have been a reason for using xargs before (even if I don't know it :p)
And I think you forgot to remove `xargs` when using `-exec`.